rstan/rstan/R/expose_stan_functions.R

```diff
@@ -31,7 +31,7 @@ expose_stan_functions_hacks <- function(code, includes = NULL) {
   code <- gsub("stan::math::accumulator<double>& lp_accum__, std::ostream* pstream__ = nullptr){",
                "std::ostream* pstream__ = nullptr){\nstan::math::accumulator<double> lp_accum__;",
                code, fixed = TRUE)
-  code <- gsub("= nullptr", "= 0", code, fixed = TRUE)
+  code <- gsub("= nullptr)", "= 0)", code, fixed = TRUE)
```

Comment:
Is there other patterns that we need to support here, such as `"nullptr,"` or `" nullptr"`?

Reply:
I would have said no, but the test failure ubuntu is making me question that. I'll investigate and update the PR

Reply:
Can we just exclude `pstream__ = nullptr`?
```suggestion
  code <- gsub("pstream__ = nullptr", "pstream__ =nullptr", code, fixed = TRUE)
  code <- gsub("= nullptr", "= 0", code, fixed = TRUE)
  code <- gsub("pstream__ =nullptr", "pstream__ = nullptr", code, fixed = TRUE)
```